### ISSUE TITLE
Hello!

### ISSUE THREAD

**Issue by MarketingPip** [live-codes/livecodes#283]
So weird! You are the first star gazer of my component and oddly enough - I literally have been creeping your repo's the past few days and was going to reach out to see if you have any interest in collaboration / making a pull request on this project. 

[Code-Runner-Web-Component: A web component that allows you to run high level programming languages on your websites (static websites included!)](https://github.com/MarketingPipeline/Code-Runner-Web-Component) 

Not only would I be grateful - but I am sure some other people would be in the community! plus - they have values inline!

**Comment by hatemhosny** [comment]
Hi @MarketingPip ,
Thank you for reaching out

nice project!

Sure, I would welcome collaboration.
I will look for some issues there that I can help with.

**Comment by MarketingPip** [comment]
@hatemhosny sorry for delay been stupid busy working on some stuff related + some other library's I'm pushing. (Currently un-employed / no client work etc) 

But hoping to adding you to a view other repos as well to get your input - as they relate to this. Plus I think I got something that might / might not be of use to you relating in next week ish for this repo. 😊 

I did get a move on doing a basic tabs function (as listed in issue - but not anything as advanced as I'm looking to do). 

Tho as said - hoping I can drop you into a repo so we can chat discussion there / or maybe even get some contact details and discuss project from there.
